Anisotropic cosmic rays (#132)
enzo-project/enzo-dev: doc/manual/source/parameters/index.rst

    Switches on diffusion of the cosmic ray energy density. Default: 0

    0. Off
    1. On; Isotropic diffusion with constant coefficient (``CRkappa``). Compatible with HydroMethod = 2 or 4.
    2. On; Anisotropic diffusion around magnetic field lines with constant coefficient (``CRkappa``). Compatible with HydroMethod = 4. 

``CRkappa`` (external)
    Cosmic ray diffusion coefficient in CGS units (cm^2/s), Default: 0.0. For MW-like galaxies: 1E28-3E29.

``CRStreaming`` (external)
    Switches on streaming of the cosmic ray energy density. Default: 0
   
   0. Off
   1. On; Anisotropic streaming around magnetic field lines. Compatible with HydroMethod = 4. 

``CRHeatng`` (external)

>     Models momentum transfer from streaming cosmic rays to the thermal gas. Physically, this should

Reviewer: ```suggestion
    This parameter adds heating of gas from streaming of cosmic rays. Physically, this should
```
